src/transforms/transform-pull-request.ts

```diff
@@ -75,11 +59,15 @@ const mapReviews = async (reviews: Octokit.PullsListReviewsResponse = [], gitHub
 
 	// Get GitHub user email, so it can be matched to an AAID
 	return Promise.all(reviewsReduced.map(async reviewer => {
-		const gitHubUser = await getGithubUser(gitHubInstallationClient, reviewer.login);
-		return {
-			...omit(reviewer, "login"),
-			email: gitHubUser?.email || `${reviewer.login}@noreply.user.github.com`
+		const mappedReviewer = {
+			...omit(reviewer, "login")
 		};
+		const isDeletedUser = !reviewer.login;
+		if (!isDeletedUser) {
```

Comment:
nit: could just do `if (!reviewer.login)` and add a comment explaining that `login` doesn't exist if the user has been deleted

Reply:
I prefer to make code self-explanatory and avoid comments